packages/search/addon/components/navi-search-bar.hbs

```diff
@@ -1 +1,15 @@
-{{yield}}
+{{! Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
+<BasicDropdown @tagName="div" @matchTriggerWidth={{true}} @verticalPosition="below" @class="navi-search" ...attributes as |dd|>
+  <dd.trigger class="navi-search-bar">
+    <Input @type="text" @value={{this.searchQuery}} @placeholder="Search" @class="navi-search-bar__input"
+      @keyUp={{fn this.onKeyUp dd}} />
+    <NaviIcon @icon="search" class="navi-search-bar__icon" />
+  </dd.trigger>
+  {{#if this.searchResults}}
+    <dd.content @class="navi-search-results navi-search-results--box navi-search-result-wrapper slide-fade">
+      {{#each this.searchResults as |result|}}
+        <NaviSearchResult::Wrapper @result={{result}} @closeResults={{fn this.closeResults dd}} class="navi-search-results__item navi-search-results--box"/>
```

Comment:
Can we move the each into the wrapper?

Reply:
Every result has its own wrapper component around it, this is what has the title and the box styles so the for each should be outside

Reply:
Isn't this the wrapper? https://github.com/yahoo/navi/pull/741/files#diff-ae9e88aabe469577377ce4a2e77b8f02R1

Reply:
ohh I guess it could work and make sense 👍